laravel/resources/views/schedules/resident/schedule_confirm.blade.php

```diff
@@ -1,202 +1,66 @@
 @extends('main')
 @section('content')
-	<!-- Use a hidden form to store selected schedule ids, milestones and education objectives. -->
-	<form method="POST" action="./submit">
-		<input type="hidden" name="schedule_id" value="{{ $id }}">
+	<!-- Use a hidden form to store selected schedule ids, milestones, education objectives, and anest pref. -->
+<form method="POST" action="./submit">
+	<input type="hidden" name="schedule_id" value="{{ $id }}">
 	<div class="row">
-		@if(!is_null($previous[0]))
+		@if(!is_null($previousChoices[0]))
+			<div class="col-12 col-md-6">
```

Comment:
This will change the way it displays for smaller screens. I am fine with the change though - should confirm with David as well  
![image](https://user-images.githubusercontent.com/40678479/112356430-fb164d00-8ca4-11eb-8270-a8c1a10a67a0.png)
![image](https://user-images.githubusercontent.com/40678479/112356447-fea9d400-8ca4-11eb-8a37-92f88e3f226c.png)

Reply:
good point. I forgot about mobile. What about `col-12 col-md` so it remains full width on mobile, but "dynamic" (half or full based on if there are prev selections or not) on desktop?